share/hybrid/Makefile

```diff
@@ -1,19 +1,14 @@
 #!/usr/bin/make -f
 
-PROGRAM = gpu-manager
-PROGRAM_FILES = gpu-manager.c
-CC = gcc
-CFLAGS =-g -Wall $(shell pkg-config --cflags libpci libdrm libkmod) -Ijson-parser json-parser/json.c
-LDFLAGS =$(shell pkg-config --libs libpci libdrm libkmod) -lm
+PROG = gpu-manager
+OBJ = gpu-manager.o json-parser/json.o
 
-all: build
+CFLAGS += -Wall $(shell pkg-config --cflags libpci libdrm libkmod)
+LDLIBS += $(shell pkg-config --libs libpci libdrm libkmod) -lm
```

Comment:
For both of these lines I think the [override directive](https://www.gnu.org/software/make/manual/html_node/Override-Directive.html) should be used.